playgrounds/react/src/examples/radio-group-compatible/radio-group-compatible-controlled-playground.tsx

```diff
@@ -0,0 +1,413 @@
+import { RadioGroupCompatible, RadioGroupRef } from '@devextreme/react';
+import { ChangeEvent, useReducer, useRef } from 'react';
+
+interface PlaygroundBtnData {
+  value: string;
+  label: string;
+}
+
+interface PlaygroundBaseDomSettings {
+  focusCss: boolean;
+  tabIndex: number;
+  hoverCss: boolean;
+  activeCss: boolean;
+  accessKey: string;
+  disabled: boolean;
+  hint?: string;
+  width?: string;
+  height?: string;
+  logOnLifecycle: boolean;
+  visible: boolean;
+}
+
+interface PlaygroundState {
+  groupValue?: string;
+  buttons: PlaygroundBtnData[];
+  baseSettings: PlaygroundBaseDomSettings;
+  focused: boolean;
+  componentMounted: boolean;
+}
+
+type Actions =
+  | { type: 'setValue', groupValue?: string }
```

Comment:
**Minor**
Better use 'kind' in such cases so that it won't be confused with other things called 'type'
```suggestion
  | { kind: 'setValue', groupValue?: string }
```

Reply:
I mean using the word 'kind' for discriminator props